backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSFiles.java

```diff
@@ -27,6 +27,7 @@
 	static final String appDir = System.getenv("HOME");
 	static final String externalPath = appDir + "/Documents/";
 	static final String localPath = appDir + "/Library/local/";
+	static final String cachePath = appDir + "/Library/Caches/";

```

Comment:
This should be retrieved with from the OS by calling somethink like
`NSFileManager.getDefaultManager().getURLsForDirectory(NSSearchPathDirectory.CachesDirectory, NSSearchPathDomainMask.AllDomainsMask).get(0).getPath();` (untested)
See https://www.swiftdevcenter.com/get-document-directory-temp-directory-and-cache-directory-path-ios-swift-5-1/